### ISSUE TITLE
Can we add a config to Redwood IDE extension which uses port 8000

### ISSUE THREAD

**Issue by ajoslin103** [redwoodjs/redwood#2511]
Is there a config option for the IDE to use some other port than 8000 ?

I work on new Redwood projects and old legacy codebases that require port 8000

I can't reconfig the old legacy code away from port 8000 

So I have to keep disabling/enabling the IDE extension all the time

**Comment by cyberious** [comment]
I agree, this is an annoying thing, I have to close code or disable the plugin in order to work on my other Gatsby projects which use 8000.

**Comment by thedavidprice** [comment]
Thanks for this, all. The Extension is in the middle of a major overhaul right now. The new version will be completely rewritten — great news, but taking time. We are organizing all these Issues to address once the dust settles.

Looping in @aldonline who is lead.

**Comment by cyberious** [comment]
While I understand the need to keep a clean backlog, perhaps this just means that the Redwood JS VS Extension should be its own repository if you are unable to keep both the core framework tickets and extension tickets in a single repo.

**Comment by cyberious** [comment]
As someone who has worked in the open-source community for a long time, I understand where you would want to purge stale issues. However, if the desire is ultimately to still address them, perhaps an alternative would be to link the issues to a project and ensure they are not lost as I am sure there are ones you will want to maintain, additionally, linking in a README within the portion of the VS Code Extension project that will bring back the context as to why a particular change is needed is also helpful. Closing as `Won't fix at this time` and `see project X` for reference to this tasks leaves a better taste for engineers looking for issues to be resolved or a workaround. 

Just my 🪙 🪙
